cli/internal/test/builder/node.go

```diff
@@ -0,0 +1,117 @@
+package builder
```

Comment:
You might consider finding a different name so this is not confused with the builder that implements `docker build`, but I don't have any ideas to offer.